src/components/UI/ButtonBordered/ButtonBordered.module.css

```diff
@@ -0,0 +1,63 @@
+.ButtonBodered {
```

Comment:
.ButtonBordered should be instead of .ButtonBodered.

Reply:
Done.

Reply:
ok